myapp/app/helpers/tasks_helper.rb

```diff
@@ -1,4 +1,13 @@
 # frozen_string_literal: true
 
 module TasksHelper
+  def sort_order(column, title)
+    direction = column == sort_column && sort_direction == 'asc' ? 'desc' : 'asc'
```

Comment:
HelperからContollerの関数を実行とかできるんですね。（知らなかった）
若干可読性に欠けるので条件を囲ってあげるとかするとちょっと良くなるかもしれません！
```suggestion
    direction = (column == sort_column && sort_direction == 'asc') ? 'desc' : 'asc'
```

Reply:
コントローラでプライベートメソッド件ヘルパーみたいな書き方は可読性に欠けるような気もしています...

Reply:
そうですね。
世の中のRailsアプリケーションでもあまり見かけない気がするので他の書き方で代替できるならしたほうがいいかもしれません！
（あくまで研修なので大規模改修が必要なのであればこのままでもOK）

Reply:
このまま行かせてください！（余裕あればリファクタします）